### ISSUE TITLE
Install instructions?

### ISSUE THREAD

**Issue by ghost** [iospackix/Packix#1]
I'm running into errors when trying to setup a Packix repo server. Can we please get an install readme? I would appreciate it, and thank you.

**Comment by andrewwiik** [comment]
No Packix is not at the stage where others can host it efficiently yet.

**Comment by ghost** [comment]
Yeah, it's alright. I went ahead and @andrewwiik you on twitter and you said that. I ended up coming up with my own solution. Look forward to seeing your project complete though! 

Thanks,
